File: Assignment 2/C++/1a.cpp
#include<iostream>
using namespace std;
int main(){
double arr[]={1,2,3,4,5,6};
int i=0;
int j=0;
double k=0;
double average;
for (i=0;i<6;i++){
    for (j=0;j<6;j++){
        if((arr[j]-arr[i])>0){
            k=k+arr[j]-arr[i];
            }
        else{
            k=k+arr[i]-arr[j];
            }
        }
    j=0;
    }
average=(k/36);
cout<<"The average distance is:"<<average;
return 0;
}

File: Assignment 2/Python/1a.py
arr=[1,2,3,4,5,6]
i=0
j=0
k=0
while i<6:
    while j<6:
        k=k+abs(arr[j]-arr[i])
        j+=1
    j=0
    i+=1
average=k/36
print("The average distance is:",average)

